articles/compliance/gdpr/features-aiding-compliance/protect-user-data.md

```diff
@@ -0,0 +1,85 @@
+---
+title: GDPR Compliance: Protect and secure user data
+description: This article discusses how customers can use Auth0 to better protect and secure their user's personal data
+toc: true
+---
+# GDPR Compliance: Protect and secure user data
+
+As per article 32 of GDPR, you have to implement appropriate measures in order to ensure a level of security appropriate to the risk, including, among other things:
+
+- Τhe encryption of personal data
+- Τhe ability to ensure the ongoing confidentiality, integrity, availability and resilience of processing systems and services
+- Τhe ability to restore the availability and access to personal data in a timely manner in the event of a physical or technical incident
+
+There are several Auth0 features than can help you achieve that, like user profile encryption, brute-force protection, breached password detection, step-up authentication, and more.
+
+## Encrypt user profile information
+
+You can encrypt user information before you save them in the user profile. You can use any encryption mechanism you'd like prior to storing data in the metadata fields, or you can use the built-in [rules](/rules) template **Encrypt sensitive data in the user profile** to implement this functionality (there is also a **Decrypt sensitive data from the user profile** template).
+
+## Enable brute-force protection
+
+You can enable the brute-force protection shield in order to stop malicious attempts to access your application. Every time Auth0 detects 10 failed login attempts, into a single account, from the same IP:
+- we can send an email to the affected user, and
+- block the suspicious IP address
+
+You can enable brute-force protection , configure which actions you want to take, and customize the blocked account email, using the dashboard. 
+
+For more information and configuration steps, see [Brute Force Protection](/anomaly-detection#brute-force-protection).
+
+## Enable breached password detection
+
+The breached password detection shield helps you identity user credentials that might have been compromised in a public data breech. 
+
+Auth0 tracks large security breaches that are happening on major third party sites. If one of your users' credentials were included in a public security breech, you can take action:
+
```

Comment:
If one of your users' credentials ~were~ is included in a public security breech, you can take action and: